### PR TITLE
Improve payment component

### PR DESCRIPTION
**Pull request by alejandroVaron**
What: Improve payment component

Why: because we need the interface to show a QR code and a purchase summary

Ticket: Create payment_item component / YAZ-17

### CODE REVIEW

File: front-end-yaz/front-end-yaz/src/components/PurchaseSummary.vue [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<template>
+  <div></div>
+</template>
```
Comment (juanDuqueS) [inline front-end-yaz/front-end-yaz/src/components/PurchaseSummary.vue:3]: does this really need to stay?

**Review by juanDuqueS — DISMISSED** [review]

**Review by juanDuqueS — CHANGES_REQUESTED** [review]
merging to develop intead of payment_component branch

File: front-end-yaz/front-end-yaz/src/components/PurchaseSummary.vue (hunk 1 above)
Comment (alejandroVaron) [reply front-end-yaz/front-end-yaz/src/components/PurchaseSummary.vue:3]: yes, is required

**Review by juanDuqueS — APPROVED** [review]

**Comment by alejandroVaron** [comment]
Payment

**Review by AngelicaAgudelo — APPROVED** [review]
Ok.

**Pull request merged**